tests/readwrite/test_bipartite_edgelist.py

```diff
@@ -48,10 +48,10 @@ def test_read_bipartite_edgelist(file_string, extra_kwargs):
 
     H = xgi.read_bipartite_edgelist(filename, nodetype=int, **extra_kwargs)
     int_edgelist = [{0, 1, 2, 3}, {4}, {5, 6}, {6, 7, 8}]
-    assert [H.edges.members(id) for id in H.edges] == int_edgelist
+    assert [H.edges.members(e) for e in H.edges] == int_edgelist
```

Comment:
maybe more specific by using `edge_id` to avoid possible confusion with edge set of members?

Reply:
Good point! Changed to `eid`.